features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql

```diff
@@ -2113,18 +2113,26 @@ INSERT INTO CM_PURPOSE_CATEGORY (NAME, DESCRIPTION, TENANT_ID) VALUES ('DEFAULT'
 
 -- Start of API-MGT Tables --
 CREATE TABLE AM_SUBSCRIBER (
-            SUBSCRIBER_ID INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
-            USER_ID VARCHAR(255) NOT NULL,
-            TENANT_ID INTEGER NOT NULL,
-            EMAIL_ADDRESS VARCHAR(256) NULL,
-            DATE_SUBSCRIBED TIMESTAMP NOT NULL,
-            PRIMARY KEY (SUBSCRIBER_ID),
-            CREATED_BY VARCHAR(100),
-            CREATED_TIME TIMESTAMP,
-            UPDATED_BY VARCHAR(100),
-            UPDATED_TIME TIMESTAMP,
-            UNIQUE (TENANT_ID,USER_ID)
-)/
+    SUBSCRIBER_ID   INTEGER GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    USER_ID         VARCHAR(255) NOT NULL,
+    TENANT_ID       INTEGER      NOT NULL,
+    EMAIL_ADDRESS   VARCHAR(256) NULL,
+    DATE_SUBSCRIBED TIMESTAMP    NOT NULL,
+    PRIMARY KEY (SUBSCRIBER_ID),
+    CREATED_BY      VARCHAR(100),
+    CREATED_TIME    TIMESTAMP,
+    UPDATED_BY      VARCHAR(100),
+    UPDATED_TIME    TIMESTAMP,
+    ORGANIZATION    VARCHAR(100),
```

Comment:
Can we have ORGANIZATION here? If user coming from multiple orgs in future how to select one org here?